tests/unit/test_utils.py

```diff
@@ -68,6 +73,30 @@ def test_double_quotes_inside():
     ), test_insert_query
 
 
+def test_handle_api_request():
+
+    url = "https://api.restful-api.dev/objects"
+    headers = {"content-type": "application/json"}
+    item = {
+        "name": "test_item",
+        "data": {"color": "blue", "price": 135},
+    }
+    payload = json.dumps(item)
+
+    response_post = handle_api_request(
+        url=url, method="POST", headers=headers, data=payload
+    )
+    assert response_post.status_code == 200
```

Comment:
tip: checking for `response.ok` is more generic than for code 200 as it returns `True` for all 2xx codes.

Reply:
[✅ Changed succesful status codes in api test](https://github.com/dyvenia/viadot/pull/661/commits/95b2c37ee28c8746d3ca5c5e448a9ac04c37239f)

Reply:
Why not just use `response.ok`? It's already built into `requests`.